src/test/java/htsjdk/tribble/readers/AsciiLineReaderTest.java

```diff
@@ -49,4 +41,20 @@ public void testReadLines() throws Exception {
         assertEquals(expectedNumber, actualLines);
 
     }
+
+    @Test public void voidTestLineEndingLength() throws Exception {
+        final String input = "Hello\nThis\rIs A Silly Test\r\nSo There";
+        final InputStream is = new ByteArrayInputStream(input.getBytes());
```

Comment:
Seriously?  Valid line endings in use today are `CR`, `LF`, `CR+LF`.  `AsciiLineReader` specifically looks for those three endings for lines. The test already covers all three valid line ending (a solo CR, a solo LF and a CR+LF).

If they are in the wrong order, they will be treated as multiple lines (existing behaviour).  I.e. `foo\n\r` would be tokenized as `<foo><eol><empty><eol>`.